### ISSUE TITLE
Add more options to BitmapText

### ISSUE THREAD

**Issue by bp74** [bp74/StageXL_BitmapFont#1]
We want more options to format the bitmap font text.
Maybe we introduce a BitmapTextFormat class similar to TextFormat.
Or just some basic features as this implementation provides for CreateJS:
https://github.com/vikramarka/createjsbitmaptext

**Comment by mnordine** [comment]
Yeah, I'm not sure if this warrants `BitmapTextFormat`, I added horizontal align here: https://github.com/mnordine/StageXL_BitmapFont/commit/74a05a4ca9897eb7466bebf929c8e0141cfbe4e2
